po/es.po

```diff
@@ -125,205 +129,206 @@ msgstr "Procesador"
-msgstr ""
+msgstr "Error de memoria de 32 bits"
 
 #: hardinfo/dmi_util.c:44
 msgid "Memory Array Mapped Address"
-msgstr ""
+msgstr "Dirección asignada de matriz de memoria"
 
 #: hardinfo/dmi_util.c:45
 msgid "Memory Device Mapped Address"
-msgstr ""
+msgstr "Dirección asignada de dispositivo de memoria"
 
 #: hardinfo/dmi_util.c:46
 msgid "Built-in Pointing Device"
-msgstr ""
+msgstr "Dispositivo apuntador embebido"
 
 #: hardinfo/dmi_util.c:47
 msgid "Portable Battery"
-msgstr ""
+msgstr "Batería portátil"
 
 #: hardinfo/dmi_util.c:48
 msgid "System Reset"
-msgstr ""
+msgstr "Reinicio del sistema"
 
 #: hardinfo/dmi_util.c:49
 msgid "Hardware Security"
-msgstr ""
+msgstr "Seguridad del hardware"
 
 #: hardinfo/dmi_util.c:50
 msgid "System Power Controls"
-msgstr ""
+msgstr "Controles de energía del sistema"
 
 #: hardinfo/dmi_util.c:51
 msgid "Voltage Probe"
-msgstr ""
+msgstr "Prueba de voltaje"
 
 #: hardinfo/dmi_util.c:52
 msgid "Cooling Device"
-msgstr ""
+msgstr "Dispositivo de refrigeración"
 
 #: hardinfo/dmi_util.c:53
 msgid "Temperature Probe"
-msgstr ""
+msgstr "Prueba de temperatura"
 
 #: hardinfo/dmi_util.c:54
 msgid "Electrical Current Probe"
-msgstr ""
+msgstr "Prueba de corriente eléctrica"
 
 #: hardinfo/dmi_util.c:55
 msgid "Out-of-band Remote Access"
-msgstr ""
+msgstr "Acceso remoto fuera de banda"
 
 #: hardinfo/dmi_util.c:56
 msgid "Boot Integrity Services"
-msgstr ""
+msgstr "Servicios de integridad del arranque"
 
 #: hardinfo/dmi_util.c:57
 msgid "System Boot"
-msgstr ""
+msgstr "Sistema de arranque"
 
 #: hardinfo/dmi_util.c:58
 msgid "64-bit Memory Error"
-msgstr ""
+msgstr "Error de memoria de 64 bits"
 
 #: hardinfo/dmi_util.c:59
 msgid "Management Device"
-msgstr ""
+msgstr "Dispositivo de gestión"
 
 #: hardinfo/dmi_util.c:60
 msgid "Management Device Component"
-msgstr ""
+msgstr "Componente de dispositivo de gestión"
 
 #: hardinfo/dmi_util.c:61
 msgid "Management Device Threshold Data"
 msgstr ""
 
 #: hardinfo/dmi_util.c:62
 msgid "Memory Channel"
-msgstr ""
+msgstr "Canal de memoria"
 
 #: hardinfo/dmi_util.c:63
 msgid "IPMI Device"
-msgstr ""
+msgstr "Dispositivo IPMI"
 
 #: hardinfo/dmi_util.c:64
 msgid "Power Supply"
-msgstr ""
+msgstr "Fuente de alimentación"
 
 #: hardinfo/dmi_util.c:65
 msgid "Additional Information"
-msgstr ""
+msgstr "Información adicional"
 
 #: hardinfo/dmi_util.c:66
 msgid "Onboard Device"
-msgstr ""
+msgstr "Dispositivo embebido"
 
 #: hardinfo/dmi_util.c:226
 msgid "Invalid chassis type (0)"
-msgstr "Clasificación maquina invalida (0)"
+msgstr "Tipo de chasis no válido (0)"
 
 #: hardinfo/dmi_util.c:227 hardinfo/dmi_util.c:228
 msgid "Unknown chassis type"
-msgstr "Clasificación mauqina desconocida"
+msgstr "Tipo de chasis desconocido"
 
 #: hardinfo/dmi_util.c:229
 msgid "Desktop"
-msgstr "PC Escritorio"
+msgstr "Ordenador de escritorio"
 
 #: hardinfo/dmi_util.c:230
 msgid "Low-profile Desktop"
-msgstr "PC economico"
+msgstr "Ordenador de bajo rendimiento"
```

Comment:
This are the "already class resulst" so "low profile" (bajo rendimiento) are the right enought!

Reply:
Do you mean replace "Ordenador de bajo rendimiento" with "Bajo rendimiento"?

Reply:
> Do you mean replace "Ordenador de bajo rendimiento" with "Bajo rendimiento"?

Yes.. due some misctranslation concepts!

Reply:
Ok.